src/Raven.Client/Constants.cs

```diff
@@ -376,6 +378,7 @@ private Facet()
                 internal class Fields
                 {
                     internal const string PowerBIJsonFieldName = "json()";
+                    internal const BindingFlags FieldsReflectionBindingFlags = BindingFlags.Instance | BindingFlags.Public;
```

Comment:
The constants we have in this file are for different purpose. Note we mostly have string constants here, most of them is public.

Since this is very internal detail of our implementation and it's related to querying and reflection I wonder if `ReflectionUtil` couldn't be a better location for this. You can do something like:

```
internal static class ReflectionUtil
{
	internal static class BindingFlagsConstants
	{
		internal const BindingFlags QueryingFields = BindingFlags.Instance | BindingFlags.Public;
	} 
```

Reply:
Ok